File: app/src/main/java/com/itsvks/layouteditor/fragments/ui/AboutFragment.java
package com.itsvks.layouteditor.fragments.ui;

import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.TextView;

import androidx.annotation.NonNull;
import androidx.annotation.Nullable;
import androidx.fragment.app.Fragment;
import androidx.lifecycle.ViewModelProvider;

import com.itsvks.layouteditor.BuildConfig;
import com.itsvks.layouteditor.databinding.FragmentAboutBinding;
import com.itsvks.layouteditor.R.string;

public class AboutFragment extends Fragment {

    private FragmentAboutBinding binding;

    @Override
    public View onCreateView(
            @NonNull LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
        binding = FragmentAboutBinding.inflate(inflater, container, false);
        return binding.getRoot();
    }

    @Override
    public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceState) {
        super.onViewCreated(view, savedInstanceState);

        binding.version.setText(getString(string.app_name) + " " + BuildConfig.VERSION_NAME);
    }

    @Override
    public void onDestroyView() {
        super.onDestroyView();
        binding = null;
    }
}

File: app/src/main/java/com/itsvks/layouteditor/fragments/ui/AboutFragment.kt
package com.itsvks.layouteditor.fragments.ui

import android.annotation.SuppressLint
import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.fragment.app.Fragment
import com.itsvks.layouteditor.BuildConfig
import com.itsvks.layouteditor.R.string
import com.itsvks.layouteditor.databinding.FragmentAboutBinding
import com.itsvks.layouteditor.utils.Constants

class AboutFragment : Fragment() {
  private lateinit var binding: FragmentAboutBinding

  override fun onCreateView(
    inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?
  ): View {
    binding = FragmentAboutBinding.inflate(inflater, container, false)
    return binding.root
  }

  @SuppressLint("SetTextI18n")
  override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
    super.onViewCreated(view, savedInstanceState)

    binding.details.text =
      getString(string.share_description, Constants.GITHUB_URL)
    binding.version.text = "v${BuildConfig.VERSION_NAME}"
  }
}
